examples/holistic_landmarker/android/app/src/main/java/com/google/mediapipe/examples/holisticlandmarker/MainViewModel.kt

```diff
@@ -0,0 +1,135 @@
+package com.google.mediapipe.examples.holisticlandmarker
+
+/*
+ * Copyright 2024 The TensorFlow Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class MainViewModel : ViewModel() {
+    val helperState: MutableLiveData<HelperState> = MutableLiveData(
+        HelperState()
+    )
+
+    fun setMinFaceLandmarkConfidence(confidence: Float) {
+        val currentLandmarkConfidence =
+            helperState.value?.minFacePresenceThreshold ?: 0f
+        helperState.value = helperState.value?.copy(
+            minFacePresenceThreshold = ((currentLandmarkConfidence.toBigDecimal() + confidence.toBigDecimal()).toFloat()).coerceIn(
```

Comment:
'set min facelandmarker confidence' shouldn't accept a delta - a set should set the value to whatever is passed to it. Let's only pass in the new value to these methods and set it rather than using the addition logic.